moh-hni-bathl7/bathl7.bat

```diff
@@ -5,7 +5,7 @@ echo -----------------------------------------
 echo *** BatchHL7 - HL7 Test Facility Starting: ***
 echo -----------------------------------------
 
-java -jar moh-hni-bathl7-1.0.0-SNAPSHOT.jar %*
+java -jar moh-bathl7.jar %*
```

Comment:
renamed the jar file to be something simple (avoid the risk of the bat file failing if the version changes)